src/schema/annotation.yaml

```diff
@@ -53,18 +53,18 @@ classes:
         range: string ## make this an object
         required: true
       type:
-        range: ontology class
+        range: OntologyClass
         description: >-
           A type from the sequence ontology
         slot_uri: rdf:type
       start:
-        is_a: gff coordinate
+        is_a: gff_coordinate
         description: The start of the feature in positive 1-based integer coordinates
         required: true
         close_mappings:
           - biolink:start_interbase_coordinate
       end:
-        is_a: gff coordinate
+        is_a: gff_coordinate
```

Comment:
Why not GffCoordinate?

Reply:
Ignore my comment thought I was looking at classes

Reply:
What did change in that line though?  Is it whitespace or something?

Reply:
Thanks for looking @scanon. Please take a look at the [initial comment for this PR](https://github.com/microbiomedata/nmdc-schema/pull/457#issue-1370608167). Let me know if that doesn't answer your question.